library/src/scala/quoted/matching/package.scala

```diff
@@ -0,0 +1,21 @@
+package scala.quoted
+
+package object matching {
+
+  /** Find an implicit of type `T` in the current scope given by `qctx`.
+   *  Return `Some` containing the expression of the implicit or
+   * `None` if implicit resolution failed.
+   *
+   *  @tparam T type of the implicit parameter
+   *  @param tpe quoted type of the implicit parameter
+   *  @param qctx current context
+   */
+  def searchImplicitExpr[T] given (tpe: Type[T], qctx: QuoteContext): Option[Expr[T]] = {
```

Comment:
One minor confusion with the name `searchImplicitExpr`: on a first look, it seems to mean to look for implicits of the type `Expr[T]`.

Reply:
How else could I call it? I could also use `searchImplicit` if the `Reflection.searchImplicit` is renamed. Then what should I call that one? Could be `searchImplicitArg`

Reply:
Can they both be called `searchImplicit`?